JobConfig/cosmic/genconfig_cosmic_general.txt

```diff
@@ -0,0 +1,27 @@
+//
+// Configuration for CosmicDYB
+// (general simulation)
+//
+// Ralf Ehrlich, 2015
+
+bool conversionGun.do = false;
+bool cosmicDYB.do = true;
+bool picapture.do = false;
+bool ejectedProtonGun.do = false;
+bool decayinorbitGun.do = false;
+bool piEplusNuGun.do = false;
+
+double cosmicDYB.mean =  -1.1; 
+double cosmicDYB.muEMin = 0.5e3;
+double cosmicDYB.muEMax = 5000e3;
+double cosmicDYB.dx     = 10000;
```

Comment:
Do these geometric parameters have any relation to the 'CrvBox' parameters?  Does one have to nest or otherwise correspond to the other?

Reply:
No, these values are for the production planes / production box of the Daya Bay cosmic ray generator. The other CrvBoxes in the fcl files are used to define G4 energy cuts.